torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8327,17 +8327,32 @@ def sample_inputs_multi_head_attention_forward(opinfo, device, dtype, requires_g
 
         yield SampleInput(q, args=sample_args, kwargs=sample_kwargs)
 
+
 # Includes some values such that N * N won't be a multiple of 4,
 # which should ensure we test the vectorized and non-vectorized
 # kernel code paths.
 NUM_SIZE0_TENSORS = 10000
 foreach_num_tensors = [20, 23] if not TEST_WITH_SLOW else [23, 30, 300]
+_foreach_inputs_default_kwargs = {"noncontiguous": False, "same_size": False, "low": None, "high": None}
```

Comment:
What does low and high refer to here?

Reply:
arguments of [`make_tensor`](https://github.com/pytorch/pytorch/blob/42d60d012e783ade919952f1f6e1a512bcb3e803/torch/testing/_creation.py#L28)